.github/workflows/DeployWiki.yml

```
@@ -0,0 +1,26 @@
+# https://github.com/marketplace/actions/github-wiki-action
+name: Deploy Wiki
+
+on:
+  push:
+    paths:
+      # Trigger only when wiki directory changes
+      - "wiki/**"
+    branches:
+      # And only on master branch
+      - main
+
+jobs:
+  deploy-wiki:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Pode ser atualizado para a versão 3, mas é cagativo
```suggestion
      - uses: actions/checkout@v3
```